app/javascript/components/QuestionnaireModal/index.js.rej

```diff
@@ -0,0 +1,7 @@
+diff a/app/javascript/components/QuestionnaireModal/index.js b/app/javascript/components/QuestionnaireModal/index.js	(rejected hunks)
```

Comment:
this too

Reply:
nope, just removed that and the other .rej file